DataAccessLayer/Crud/CategoryCrud.cs

```diff
@@ -24,8 +25,7 @@ public List<Category> GetAll()
             }
             catch (Exception e)
             {
-                Console.WriteLine($"\nClassName - {this.GetType().Name}\nMethodName - {MethodBase.GetCurrentMethod().Name}" +
-                                  $"\nException message: {e.Message}. \nException stack trace: {e.StackTrace}");
+                HandleExceptions.ShowInConsole(this.GetType().Name, MethodBase.GetCurrentMethod().Name, e);
```

Comment:
Not the best class naming convention, you should avoid using werbs in names - instead you can rename it to somethign like "ExceptionHandler"